package-lock.json

```diff
@@ -1,5 +1,14 @@
 {
   "name": "cmr-stac",
   "version": "1.0.0",
-  "lockfileVersion": 1
+  "lockfileVersion": 2,
```

Comment:
This is an indication that you are running a node version that is deferent from what is locked in `.nvmrc` when you run `npm install`. You should update the node version in `.nvmrc` and deployment if that is what you intended to do.